src/Framework/Environment.php

```diff
@@ -70,10 +70,9 @@ public static function setupColors()
 				&& ((function_exists('posix_isatty') && posix_isatty(STDOUT))
 					|| getenv('ConEmuANSI') === 'ON' || getenv('ANSICON') !== FALSE) || getenv('term') === 'xterm-256color');
 
-		$colors = & self::$useColors;
-		ob_start(function ($s) use (& $colors) {
-			return $colors ? $s : Dumper::removeColors($s);
-		}, PHP_VERSION_ID < 50400 ? 2 : 1, FALSE);
+		ob_start(function ($s) {
```

Comment:
This should probably stay https://github.com/nette/tester/commit/d76cd9776f757763ce1479602c1c3232c95c0a81